extensions/wikia/PortableInfoboxBuilder/tests/PortableInfoboxBuilderHelperTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+class PortableInfoboxBuilderHelperTest extends WikiaBaseTest {
+
+	protected function setUp() {
+		$this->setupFile = dirname( __FILE__ ) . '/../PortableInfoboxBuilder.setup.php';
+		parent::setUp();
+	}
+
+	/**
+	 * @dataProvider titleTextProvider
+	 */
+	public function testGetUrlPath( $titleText, $expected ) {
+		$this->assertEquals( $expected, PortableInfoboxBuilderHelper::getUrlPath( $titleText ) );
+	}
+
+	/**
+	 * @dataProvider requestModeProvider
+	 */
+	public function testForcedSourceMode( $queryStringValue, $expectedResult ) {
+		$requestMock = $this->getMockBuilder( 'WebRequest' )->setMethods( [ 'getVal' ] )->getMock();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "getVal" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)